pkgs/kona/src/cli/coverage/diff_addition_filter_merge.ts

```diff
@@ -61,7 +61,7 @@ function mergeChanges(fileDiffs: File[]) {
     const lines = Object.entries(changes)
         .map(entry => {
             const [, change] = entry
-            const line = change.type === 'normal' ? change.ln1 : change.ln
+            const line = change.type === 'normal' ? change.ln2 : change.ln
```

Comment:
What's `ln2` vs `ln1`?

Reply:
The `normal` change type is a change that doesn't add or delete any code. It's just something that has moved. `ln1` is the old line it was on and `ln2` is the new line. We want to use the new line when testing.